ignore/src/types.rs

```diff
@@ -142,6 +142,7 @@ const DEFAULT_TYPES: &'static [(&'static str, &'static [&'static str])] = &[
     ("pdf", &["*.pdf"]),
     ("php", &["*.php", "*.php3", "*.php4", "*.php5", "*.phtml"]),
     ("pod", &["*.pod"]),
+    ("powershell", &["*.ps1", "*.psd1", "*.psm1"]),
```

Comment:
Would it be better to use `ps` here? I feel like `powershell` is a bit long. Is `ps` is accepted abbreviation of PowerShell that folks would recognize?